sdk/identity/azure-identity/src/test/java/com/azure/identity/EnvironmentCredentialTests.java

```diff
@@ -36,4 +36,46 @@ public void testCreateEnvironmentCredential() {
             .expectNextMatches(token -> "token".equals(token.getToken()))
             .verifyComplete();
     }
+
+    @Test
+    public void testCreateEnvironmentClientCertificateCredential() {
+        Configuration configuration = Configuration.getGlobalConfiguration();
```

Comment:
Could remove the local instance since `Configuration` uses the fluent pattern for `put`.